frontend/src/scenes/paths/Paths.js

```diff
@@ -283,8 +284,10 @@ export class Paths extends Component {
                             <NoData />
                         ) : (
                             !dataLoaded && (
-                                <div className="loading-overlay" style={{ paddingTop: '14rem' }}>
+                                <div style={{ paddingTop: '14rem', textAlign: 'center' }}>
                                     <div />
+                                    <Spin />
```

Comment:
There's no loader when changing the date. willing to accept out of scope for this PR

Reply:
That scene could use a good refactor :). I'd say it's outside the scope of this PR, as nothing changed on Paths from before, except just the addition of the spinner on the (shown-once) loading screen.